### ISSUE TITLE
Board Not working

### ISSUE THREAD

**Issue by abhi-84** [sipeed/MaixPy-v1#448]
Hi there

**Describe the bug**
Maix Duino Board is not connecting with Maixpy IDE and if connected, not working as expected. 
Initially, everything was working fine and I was able to do mobile net, yolo and few other models. 

Then in order to to access gpio pins of board, I have added config files of Maixduino as given here [https://wiki-sipeed-com.translate.goog/soft/maixpy/zh/get_started/board_info.html?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en-GB&_x_tr_pto=nui](url)

Still I could not access gpio pins but now, after doing this, board is not connecting to IDE with firmware "Maixpy IDE with maixpy_v0.6.2_72_g22a8555b5_minimum_with_ide_support.bin". It shows a blank screen, even after multiple flashes.

When I use "maixpy_v0.6.2_72_g22a8555b5.bin" firmware, it connects properly and video streaming happens but when I try to load the model, it throws "OSError: [Errno 19] ENODEV" error. Looks like sd card is not available as a message of SD card available does not show on LCD. I have also formatted sd card multiple times with the official tool with no success.

I have reinstalled IDE and redownloaded binary images but it does not help

Please help!

**Please complete the following information**
 - IDE version: [e.g. 0.2.5] --0.2.5
 - Firmware version: [e.g. 0.6.2]--0.6.2
 - Board: [e.g. Maix Amigo]-- maix duino
 - OS: [e.g. Windows/Linux/ios]--Linux

**Comment by abhi-84** [comment]
Look slike something goes wrong with json file for configuration. How to remove that config file from flash?

**Comment by abhi-84** [comment]
I think I have managed to resolve the issue. I have made "partial delete" using kflash gui tool.
Select Erase-->Mode (Partial Erase)-->Template (Maixpy FS)-->Load--Select correct port and hit ERASE.
This will erase flash filesystem and all files placed in /flash. This helped me get rid of the config file

Now SD card is getting detected and the minimal image is also working as expected!
Hope it helps someone who get stuck in similar situation
